Auth.org

```diff
@@ -0,0 +1,66 @@
+#+TITLE: Clio Authentication and Authorization
```

Comment:
What is this file? It looks like documentation but if so I'm not sure why it's called `Auth.org` and is stored at the root of the clio repo.